src/dag/chunk.ts

```diff
@@ -67,3 +67,19 @@ export function defaultChunkHasher<V extends Value>(data: V): Hash {
 }
 
 export type ChunkHasher = typeof defaultChunkHasher;
+
+export function makeTestChunkHasher(prefix = 'fake'): ChunkHasher {
```

Comment:
This overlaps with https://github.com/rocicorp/replicache/pull/765, and will go away once that is landed and this is rebased.